build-recipe-spec

```diff
@@ -167,6 +167,18 @@ recipe_build_spec() {
     if test -n "$RUN_SHELL"; then
 	chroot $BUILD_ROOT su -
     else
+
+	# fix su imposed path for newer rhel variants
+	if [ -e "/etc/redhat-release" ];then
```

Comment:
This won't work reliably. The entire extended Red Hat family provides this file.